R/utils.R

```diff
@@ -167,4 +167,36 @@ globalVariables(
     "locations", "n", "quantile", "region", "target", "target_end_date",
     "type", "value", ".", "primary", "secondary", "target_date", "variable"
   )
-)
\ No newline at end of file
+)
+
+
+
+#' Authentificate for Google Sheets
+#'
+#' @param service_account the path to a JSON file that has all the information
+#' of the Google service account. If a service account is presented it will 
+#' be used and the other arguments will be ignored
+#' @param enail alternatively, provide the email address to an authorised 
+#' account
+#' @param cache_folder path to the folder where secrets for the email address
+#' provided are stored. 
+#' @importFrom here here
+#' @export
+#' @return NULL
+google_auth <- function(service_account = "default", 
+                        email = "epiforecasts@gmail.com", 
+                        cache_folder = ".secrets") {
```

Comment:
service account as arg?

Reply:
I could just have this function without any arguments and have the defaults hard-coded. Happy to change if you think that is better

Reply:
ah sorry I didn't realise default led to using the string as listed. This is good.